docs/release_notes/v0.16.0.rst

```diff
@@ -0,0 +1,30 @@
+SMQTK-Classifier Pending Release Notes
+======================================
+
```

Comment:
Add a short description to the release. See the other package v0.16.0 release notes to get a sense.